server/service/src/processors/transfer/shipment/create_inbound_shipment.rs

```diff
@@ -66,6 +67,19 @@ impl ShipmentTransferProcessor for CreateInboundShipmentProcessor {
         if linked_shipment.is_some() {
             return Ok(None);
         }
+        // 5.
+        let store = StoreRowRepository::new(connection)
+            .find_one_by_id(&record_for_processing.other_party_store_id)?
+            .ok_or(RepositoryError::NotFound)?;
+
+        let store_created_datetime = store
+            .created_date
```

Comment:
This might be nicer, e.g. you don't need to wrap `invoice_created_datetime` in a Some():

```rust
if let Some(created_date) = store.created_date {
   ...
}
```